api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -516,6 +516,19 @@ public void getPeople_shouldMatchSearchToFamilyName2() throws Exception {
 		assertTrue(TestUtil.containsId(people, 5));
 	}
 	
+	@Test
+	public void getPeople_shouldMatchSearchToFamilyName2DeadAndVoided() throws Exception {
```

Comment:
seems like you just copied an existing test for `getPeople(string, boolean)` and just added another parameter.

Try to figure out the different cases that the existing code can be called/go through. This is what we should test. Also think how you can improve on existing tests instead of just copying them.

you can for example test

* getPeople_shouldMatchSearchToFamilyName2DeadAndIncludeVoided

setting voided to true

Reply:
@teleivo Ok I will try with new tests. Greatful for your advice.